npm/postinstall.ts

```diff
@@ -42,18 +45,36 @@ const platformMap: Partial<
  * Fetches the supposed URL of the binary on GitHub for this version.
  */
 async function getBinaryUrl(): Promise<URL> {
-  const version = process.env.npm_package_version?.trim();
+  const debug = rootDebug.extend("getBinaryUrl");
+
+  const { arch, platform } = getArchPlatform();
+
+  debug({ arch, platform });
+
+  let version = process.env.npm_package_version?.trim();
+  debug("npm_package_version:", version);
+  /**
+   * Windows builds are currently not working past v0.14.6, so let's pin to that
+   * version here if we're on Windows.
+   */
+  if (platform.platform === "windows") {
+    debug("Windows detected; pinning to v0.14.6");
+    console.warn(
+      "Windows detected; pinning to last known working version 0.14.6"
+    );
+    version = "0.14.6";
```

Comment:
I think this works for MVP, we could add something in the future that hits the open rest API endpoint (`https://api.github.com/repos/inngest/inngest/releases`) and iterates through and warns if there isn't an asset for that resource and if not, attempts to download the latest release for that platform. Definitely more work though, but could be more future proof after this hotfix.